src/components/InputBooks.css

```diff
@@ -0,0 +1,52 @@
+.addBookHeader {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 6.25rem;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+}
+
+input.title {
+  width: 37.813rem;
+  height: 2.813rem;
+  margin: 0 2.125rem 2rem 6.25rem;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+input.author {
+  width: 17.813rem;
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0;
+  padding: 0.375rem 0.938rem 0.563rem 1rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.addBook {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  outline: none;
+  background-color: #0290ff;
+  color: #fff;
+}
+
+.addBook:hover {
+  border: none;
+  background-color: antiquewhite;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.2);
+  color: #0290ff;
+}
+
+.addBook:active {
+  border-bottom: 2px solid rgb(250, 166, 56);
```

Comment:
- The Input section of your project does not exactly look like te design template, remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
- Please pay attention to the sections I have marked in your project and ensure they are styled as it is in the design template 👍 👍 
<img width="1006" alt="f34f34f" src="https://user-images.githubusercontent.com/73607512/214806816-6637e461-956e-476c-b13a-87a92544938e.PNG">

Design template:
<img width="525" alt="tevetv" src="https://user-images.githubusercontent.com/73607512/214806995-b55a9424-90ba-41e4-90d8-f65c0c2fae53.PNG">